src/eduid_webapp/email/views.py

```diff
@@ -150,7 +150,7 @@ def post_primary(user, email, csrf_token):
 
 
 def _steal_mail(email):
```

Comment:
The "stealing" is now done before a write to the central db, https://github.com/SUNET/eduid-am/blob/master/eduid_am/consistency_checks.py.